cgp/ea/mu_plus_lambda.py

```diff
@@ -34,8 +35,15 @@ def __init__(
         ----------
         n_offsprings : int
             Number of offspring in each iteration.
+<<<<<<< HEAD
```

Comment:
hmm, something went wrong here

Reply:
oops didn't check the docstring while merging